src/NCronJob/NCronJobExtensions.cs

```diff
@@ -60,6 +74,16 @@ public static IServiceCollection AddNCronJob(
         services.TryAddSingleton<MissingMethodCalledHandler>();
 
         return services;
+
+        void Feed(JobDefinition jobDefinition, List<DependentJobRegistryEntry> dependentJobs)
+        {
+            jobDefinitions.TryAdd(jobDefinition, []);
```

Comment:
This whole method might be easier like this?
```csharp
var entries = jobDefinitions.GetOrCreate(jobDefinition);
entries.AddRange(dependentJobs);
```

WHere `GetOrCreate` is almost like your `AddInto`:
```csharp
internal static class DictionaryExtensions
{
    public static TCollection GetOrCreate<TKey, TCollection, TValue>(
        this IDictionary<TKey, TCollection> dic, 
        TKey key
    ) where TKey : notnull
      where TCollection : ICollection<TValue>, new()
    {
        if (!dic.TryGetValue(key, out var entries))
        {
            entries = new TCollection();
            dic[key] = entries;
        }
        return entries;
    }
}
```

Reply:
That works, but the compiler cannot infer the type of `TValue`. Which requires to specify all the type parameters

```csharp
var entries = jobDefinitions.GetOrCreate<JobDefinition, List<DependentJobRegistryEntry>, DependentJobRegistryEntry>(jobDefinition);
entries.AddRange(dependentJobs);
```

vs

```csharp
jobDefinitions.AddInto(jobDefinition, dependentJob);
```

@linkdotnet Which one do you prefer?

Reply:
Hmm that is fair because of the `List` - if we do something like this:
```csharp
public static List<TValue> GetOrCreateList<TKey, TValue>(
    this IDictionary<TKey, List<TValue>> dic,
    TKey key
) where TKey : notnull
{
    if (!dic.TryGetValue(key, out var entries))
    {
        entries = new List<TValue>();
        dic[key] = entries;
    }
    return entries;
}
```

That should work, shouldn't it?

Reply:
Fixed